src/site/content/en/progressive-web-apps/pwa-checklist/index.md

```diff
@@ -0,0 +1,482 @@
+not change with viewport size. The content can be rearranged at different
+viewport sizes, and it should all be there, one way or another. In fact, as
+Luke Wroblewski states in his book Mobile First, starting small and going
+large instead of the other way around can actually improve a site's design:
+
+> Mobile devices require software development teams to focus on only the
+> most important data and actions in an application. There simply isn't room
+> in a 320 by 480 pixel screen for extraneous, unnecessary elements.
+> You have to prioritize.
+>
+> So when a team designs Mobile First, the end result is an experience focused
+> on the key tasks users want to accomplish without the extraneous detours
+> and general interface debris that litter today's desktop-accessed Web sites.
+> That's good user experience and good for business.
+
+#### How
+
+There are many  resources on responsive design, including the
+[original article by Eathan Marcotte](https://alistapart.com/article/responsive-web-design/),
+a [collection of important concepts](https://snugug.com/musings/principles-responsive-web-design/)
+related to it, as well as books and talks galore. To narrow this discussion
+down to the content aspects of responsive design, you can dig into
+[content-first design](https://uxdesign.cc/why-you-should-design-the-content-first-for-better-experiences-374f4ba1fe3c)
+and [content-out responsive layouts](https://alistapart.com/article/content-out-layout/).
+Finally, while it's focused on mobile, the lessons in
+[Seven Deadly Mobile Myths](https://www.forbes.com/sites/anthonykosner/2012/05/03/seven-deadly-mobile-myths-josh-clark-debunks-the-desktop-paradigm-and-more/#21ecac977bca)
+by Josh Clark are just as relevant to small-sized views of responsive sites
+as they are to mobile.
+
+{% endDetails %}
+
+
+{% Details %}
+{% DetailsSummary 'h3' %}
+
+Provides a custom offline page
+
+When users are offline, keeping them in your PWA provides a more seamless
+and native-like experience than dropping back to the default browser offline
+page.
+
+{% endDetailsSummary %}
+
+When users are offline, keeping them in your PWA provides a more seamless
+and native-like experience than dropping back to the default browser offline
+page.
+
+#### Why
+
+Users expect installed apps to work regardless of their connection status.
+A native app never shows a blank page when it is offline, and a Progressive
+Web App should never show the browser default offline page. Providing a
+custom offline experience, both when a user navigates to a URL that hasn't
+been cached and when a user tries to use a feature that requires a connection,
+helps to bridge the native-web app gap and makes your experience feel more
+native.
+
+#### How
+
+During a service worker's `install` event, you can precache a custom offline
+page for later use. If a user then goes offline and your service worker is
+unable to fetch an HTML page, you can respond with precached custom offline
```

Comment:
"If a user then goes offline and your service worker is unable to fetch an HTML page, you can respond with precached custom offline page."

Shorten to:

"If a user goes offline, you can respond with the precached custom offline page."